src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -373,3 +373,5 @@ function resizeCrudTableColumnWidths() {
   </script>
 
   @include('crud::inc.details_row_logic')
+  
+  @include('crud::inc.error_frame')
```

Comment:
Why limit this feature to DataTables only? This would be useful in ALL Backpack pages. Let's move this to bottom of the `scripts` blade file. That way, anywhere in your admin panel you have an AJAX error... you see the error. Including, but not limited to:
- in AJAX charts (dashboard)
- when using `InlineCreate` or `select2_from_ajax` in forms (Create/Update operations)

Please test those scenarios above too, just to make sure they actually help there. But it should "just work", I think.

Reply:
Makes total sense!
This is now working with everything 👌